src/components/MessagesList/MessagesGroup/Message/MessageItem.vue

```diff
@@ -51,8 +51,14 @@
 				'bottom-side': isSplitViewEnabled && !isShortSimpleMessage && (isSmallMobile || isSidebar),
 				overlay: isSplitViewEnabled && !isShortSimpleMessage && isReactionsMenuOpen && !(isSmallMobile || isSidebar),
 			}">
+			<ScheduledMessageActions
+				v-if="showScheduledMessages && showMessageButtonsBar"
+				v-model:is-action-menu-open="isActionMenuOpen"
+				:message="message"
+				class="message-buttons-bar"
```

Comment:
You should use both classes

```suggestion
				class="message-buttons-bar"
				:class="{ outlined: buttonsBarOutlined }"
```